infrastructure/source_bundle/proxy/conf.d/application.conf

```diff
@@ -35,7 +35,7 @@ server {
                 proxy_set_header Upgrade $http_upgrade;
                 proxy_set_header Connection "Upgrade";
                 proxy_pass_request_headers on;
-                client_max_body_size 200m;
+                client_max_body_size 5m;
```

Comment:
Thanks, I think this was a leftover of another project setup where we allow to upload heavy files. No needed in this project